buildtest/buildsystem/base.py

```diff
@@ -457,13 +458,13 @@ def run_tests(self, testfile):
         if command.returncode == 0:
             print(
                 "{:<30} {:<30} {:<30} {:<30}".format(
-                    self.config_name, self.name, "PASSED", self.config_file
+                    self.name, self.name, "PASSED", self.buildspec
```

Comment:
This needs to be changed back to be the config name, then self.name, which is the key in the file.